src/components/ConfigAnalysis/ConfigAnalysisSeverityDropdown.tsx

```diff
@@ -3,18 +3,42 @@ import { Control } from "react-hook-form";
 import {
   HiExclamation,
   HiExclamationCircle,
-  HiInformationCircle
+  HiInformationCircle,
+  HiOutlineChevronDoubleDown,
+  HiOutlineChevronDown,
+  HiOutlineMinus
 } from "react-icons/hi";
 import { defaultSelections } from "../Incidents/data";
 import { ReactSelectDropdown } from "../ReactSelectDropdown";
 
 export const configAnalysisSeverityItems = {
+  Low: {
```

Comment:
Can we not reuse the same constants as the incident severities ?

Reply:
In config insights, we seem to have an extra severity - info, that doesn't exist in incidents severity, other than that, we can.